src/index.ts

```diff
@@ -165,6 +170,11 @@ export class ServerlessClamscan extends Construct {
    */
   public readonly defsAccessLogsBucket?: IBucket;
 
+  /**
+    Conditional: When true, the user accepted the responsibility for using external buckets
+   */
+  public readonly useExternalBuckets?: boolean;
```

Comment:
```suggestion
  public readonly useImportedBuckets?: boolean;
```

Reply:
is it imported the right term? It makes me thing that somehow the bucket has been imported into the cloudformation template as resource, but it's not the case, only a reference to the bucket is used internally